src/lib/AutocompleteTextInput.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import { FormElement } from "./index.js";
+  import { v4 as uuidv4 } from "uuid";
+
+  export let label;
+  export let name;
+  export let value;
+  export let options;
+  export let dataTestId = "";
+
+  let id = `${name}-${uuidv4()}`;
+</script>
+  
+<FormElement {label} id={`${name}-${id}-input`}>
+  <input
+    class="govuk-input govuk-input--width-20"
+    id={`${name}-${id}-input`}
+    data-testid={dataTestId}
+    list={`${name}-${id}-list`}

```

Comment:
```suggestion
    {id}
    list={`${id}-list`}
```

Don't need `data-testid`, it's for helping Playwright and is a last resort anyway (supposed to use locators that the user can see)

Reply:
Sorry only just saw this, understood